app.rb

```diff
@@ -1,8 +1,40 @@
-require 'sinatra/base'
+require "sinatra"
+require 'sinatra/reloader' if development?
+require 'pg'
 
 class Chitter < Sinatra::Base
-  get '/test' do
-    'Test page'
+
+  configure :development do
+    register Sinatra::Reloader
+  end
+
+  get "/" do
+    redirect :messages
+  end
+
+  get "/messages" do
+    @reversed = params[:reversed]
+    @find = params[:find]
+    connection = PG.connect(dbname: 'chitter_test', user: 'JMMakers', password: '1234')
```

Comment:
We could use environment variable so chitter_test is only used in the test environment and chitter is used in production environment.